### ISSUE TITLE
Qt client: tts events for transmition toggles

### ISSUE THREAD

**Issue by amirmahdifard** [BearWare/TeamTalk5#1651]
### Description

@CoBC so i see that you're adding some interesting and wunderful tts events to Qt client, for example the one that says subscription voice enabled for amir: so also, please add a tts event for toggeling users transmitions: for example, if i press alt controle q on someones name, it should say, like, trasmition voice disabled for amirmahdifard: or, amirmahdifard's voice transmition voice disabled: or something like that: currently, we only have one tts event for transmitions toggle, wich says amirmahdifard can no longer transmit voice: but please also add this one too like subscriptions, because it will be useful for example if you are admin and you want to press alt controle q on someones name wich is in another channel and not in your channel:  and it looks handy generally: qt client vertion: 5.11

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by ChrisDuffley** [comment]
Please see https://github.com/BearWare/TeamTalk5/pull/1460 as this is already enabled for classroom changes in TTS settings.

**Comment by amirmahdifard** [comment]
@ChrisDuffley no, this is difrent, for subscriptions we have 2 tts events, so y not also in transmitions?

**Comment by ChrisDuffley** [comment]
It sounds like to avoid duplicated TTS events you may want https://github.com/BearWare/TeamTalk5/pull/1507 reverted?
